plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestingExasolServer.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.exasol;
+
+import com.exasol.containers.ExasolContainer;
+import com.exasol.containers.ExasolService;
+import com.exasol.jdbc.EXADriver;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.jdbc.BaseJdbcConfig;
+import io.trino.plugin.jdbc.ConnectionFactory;
+import io.trino.plugin.jdbc.DriverConnectionFactory;
+import io.trino.plugin.jdbc.RetryingConnectionFactory;
+import io.trino.plugin.jdbc.credential.StaticCredentialProvider;
+import io.trino.testing.ResourcePresence;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import java.util.Map;
+
+import static io.trino.testing.TestingConnectorSession.SESSION;
+import static java.lang.String.format;
+
+public class TestingExasolServer
+        implements Closeable
+{
+    public static final String TEST_USER = "trino_test";
+    /**
+     * Name of the test schema. Must not contain an underscore, required by {@link io.trino.plugin.exasol.TestExasolConnectorTest#testShowSchemasLikeWithEscape()}
+     */
+    public static final String TEST_SCHEMA = "trinotestschema";
+    public static final String TEST_PASS = "trino_test_password";
+
+    private final ExasolContainer<?> container;
+
+    private TestingExasolServer()
+    {
+        container = startContainer();
+    }
+
+    private static ExasolContainer<?> startContainer()
+    {
+        ExasolContainer<?> container = new ExasolContainer<>().withRequiredServices(ExasolService.JDBC);
+        // Deactivate reuse to allow running test classes in parallel.
+        container.withReuse(false);
```

Comment:
Is it enabled by default? I thought the default value is false.

Reply:
You are right, it is disabled by default. I wanted to make it easy to enable it during development.

Reply:
Let's remove it then.